.github/workflows/email.yml

```diff
@@ -42,7 +42,16 @@ jobs:
         id: changed-files
         uses: tj-actions/changed-files@v34
         with:
-          separator: ","                 
+          separator: "," 
+      # tj-actions' limiation prevents the user from using "\n" as ouptput seperator.
+      # parse the list of changed files and add <br> to add line feeds.   
```

Comment:
Want to indicate this is the solution and indicate where you are adding <br>s.  "Solution: Parse the list of changed files and replace each comma delimiter with a <br>."

Reply:
fixed.